crops/core/ops.py

```diff
@@ -50,22 +50,23 @@ def get_sequence_alignment(sequence_1, sequence_2, mode='global', open_gap_score
     return alignment_dict
 
 
-def renumber_pdb_needleman(inseq, instr):
+def renumber_pdb_needleman(inseq, instr): #  , seqback=False):
     """Returns modified :class:`gemmi.Structure` with new residue numbers. It uses Needleman-Wunsch
     algorithm to perform the sequence alignment
 
     :param inseq: Input sequence.
     :type inseq: :class:`~crops.elements.sequences.oligoseq`
     :param instr: Gemmi structure.
     :type instr: :class:`gemmi.Structure`
-    :param seqback: If True, it additionally returns the :class:`~crops.elements.sequences.oligoseq` with the gaps found in the structure, defaults to False.
-    :type seqback: bool, optional
```

Comment:
Yes it makes sense to remove this, I probably copy pasted this from a different docstring